src/betterproto/plugin/plugin.bat

```diff
@@ -0,0 +1,2 @@
+@SET plugin_dir=%~dp0
+@python -m %plugin_dir% %*
```

Comment:
Not sure how this will pan out, not a .bat person...

Reply:
I will test this out when I have time in the next couple of days :+1:

Reply:
Is this used anymore?